frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/select-deployment.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { TFunction } from 'i18next';
+import { useTranslation } from 'react-i18next';
+import {
+  FormGroup,
+  Select,
+  SelectOption,
+  SelectProps,
+  SelectVariant,
+} from '@patternfly/react-core';
+
+import { DeploymentType } from '../../../../constants/create-storage-system';
+import { WizardDispatch, WizardState } from '../../reducer';
+import './backing-storage-step.scss';
+
+const options = [DeploymentType.FULL, DeploymentType.MCG];
+
+const optionsDescription = (t: TFunction) => ({
+  [DeploymentType.MCG]: t(
+    'ceph-storage-plugin~Deploys MultiCloud Object Gateway without block and file services.',
+  ),
+  [DeploymentType.FULL]: t(
+    'ceph-storage-plugin~Deploys OpenShift Data Foundation with block, shared fileSystem and object services.',
```

Comment:
![Screenshot from 2021-12-08 19-24-03](https://user-images.githubusercontent.com/57935785/145224223-457903c8-da26-4e94-b54a-e9e5ffda68b0.png)
I am unable to see this description for full deployment. Please check

Reply:
yeah I added later , you may check on latest